.cursorrules

```diff
@@ -0,0 +1,53 @@
+# WordPress.com Studio
+
+Every time you choose to apply a rule(s), explicitly state the rule(s) in the output. You can abbreviate the rule description to a single word or phrase.
```

Comment:
What is the purpose of stating he rule(s) output?
I've seen in BigSky they set the personality, something like:
You are a great Software Engineer, creating maintainable code ....